lib/l2meter/emitter.rb

```diff
@@ -4,6 +4,8 @@ module L2meter
   class Emitter
     attr_reader :configuration
 
+    BARE_VALUE_PLACEHOLDER = Object.new
```

Comment:
It probably doesn't matter because we're using it as a sentinel value, but should we `#freeze` the placeholder?

Reply:
I'm not sure what the actual effect in the ruby runtime is (eg does it optimise it somehow?) but I think it's worth doing anyway, purely from a readability point of view 👍 

Btw I was struggling to find a better term than "placeholder", but "sentinel" is better so I'll also switch it to that - thanks.

Going to make those changes and squash the branch ready for merging.